src/views/Layout.js

```diff
@@ -0,0 +1,11 @@
+import 'assets/styles/reset.css';
+import React from 'react';
+import Helmet from 'react-helmet';
+import { TITLE_TEMPLATE } from 'server/getBrowserTitle';
+
+export default ({ children }) => (
+  <div>
+    <Helmet titleTemplate={TITLE_TEMPLATE} />
```

Comment:
Here we globally define the default template for that page `<title>`, via the template exported in the `getBrowserTitle` helper.